applications/DEMApplication/custom_constitutive/DEM_D_Hertz_viscous_Coulomb_2D_CL.cpp

```diff
@@ -44,9 +47,14 @@ namespace Kratos {
         const double my_shear_modulus = 0.5 * my_young / (1.0 + my_poisson);
         const double walls_shear_modulus = 0.5 * walls_young / (1.0 + walls_poisson);
         const double equiv_shear = 1.0 / ((2.0 - my_poisson) / my_shear_modulus + (2.0 - walls_poisson) / walls_shear_modulus);
-
-        mKn = 0.7854 * equiv_young;     //KRATOS_M_PI_4 = 0.7854
+        const double element_radius = element->GetRadius();
+        mKn = 2.0 * element_radius * equiv_young;
```

Comment:
In order to avoid the future question 'why this?' we could paste here the reference of the paper or book used for this. It will be very helpful if possible.

Reply:
Thanks! Maybe @salvalatorre knows about this equation.

Reply:
This question has been solved.